sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -145,9 +147,12 @@ ENDIF(BUILD_PYTHON_INTERFACE)
 
 
 # --- CHECK EXAMPLES -----------------------------------------------------------
-ADD_EXECUTABLE(master-board-example example/example.cpp)
-TARGET_LINK_LIBRARIES(master-board-example ${PROJECT_NAME})
-INSTALL(TARGETS master-board-example DESTINATION bin)
+ADD_EXECUTABLE(master_board_example example/example.cpp)
+TARGET_LINK_LIBRARIES(master_board_example ${PROJECT_NAME})
+target_include_directories(
+    master_board_example PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
```

Comment:
Same thing here for `$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>`

Reply:
See above